anchore_engine/services/policy_engine/engine/vulns/providers.py

```diff
@@ -59,29 +54,40 @@ class VulnerabilitiesProvider:
     __scanner__ = None
     __cache_manager__ = None
 
+    @abstractmethod
     def load_image(self, **kwargs):
         """
         Ingress the image and compute the vulnerability matches. To be used in the load image path to prime the matches
         """
-        raise NotImplementedError()
+        ...
+
+    @abstractmethod
+    def get_image_vulnerabilities_json(self, **kwargs) -> json:
```

Comment:
I'm not sure 'json' is what you intended here. I think this is using the json module as the type.

Reply:
That's incorrect, not sure how it made it there